x/concentrated-liquidity/lp.go

```diff
@@ -65,6 +65,10 @@ func (k Keeper) createPosition(ctx sdk.Context, poolId uint64, owner sdk.AccAddr
 		return 0, sdk.Int{}, sdk.Int{}, sdk.Dec{}, time.Time{}, err
 	}
 
+	if sqrtPriceLowerTick.Equal(sqrtPriceUpperTick) {
+		return 0, sdk.Int{}, sdk.Int{}, sdk.Dec{}, time.Time{}, errors.New("sqrtPriceLowerTick and sqrtPriceUpperTick cannot be equal")
+	}
```

Comment:
Note: I had to make this change to make the benchmark work

Keeping it with the understanding that @czarcas7ic 's change in a separate PR will fully address and test this

Reply:
If possible, would be great to branch this off mine to just remove this entirely. But if we are time constrained this can be done later.